Charts/Classes/Renderers/CandleStickChartRenderer.swift

```diff
@@ -43,7 +43,7 @@ public class CandleStickChartRenderer: LineScatterCandleRadarChartRenderer
         }
     }
     
-    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 4)
+    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 2)
```

Comment:
Not sure what you are saying. That change is simply a type, I removed count: by accident and put it back and instead of 4, I put it back as 2. For other changes, I don't use the tool because with Swift 3 it causes of syntax issue. Also many in this beta were not suggested.